setup.py

```diff
@@ -23,6 +36,8 @@
     libdir = os.path.abspath(os.path.join(scipoptdir, 'lib/shared'))
     libname = 'scip'
     extra_compile_args.append('-DNO_CONFIG_HEADER')
+    # the following is a temporary hack to make it compile with SCIP/make:
+    extra_compile_args.append('-DTPI_NONE') # this is just a guess, if other TPIs are used, please modify
```

Comment:
If this is just a guess: can you please try if this works and paste the output? Thanks!

Reply:
This works. What kind of output do you want? I do not think that you see anything in the output ...

Reply:
I just quoted your comment ;) 
If you checked that this worked it's fine